howto/application/stream-application.md

```diff
@@ -59,9 +61,31 @@ The downloaded `.csv` file has the following statistics:
 
 ### Sharing a streaming session
 
+
+````{tabs}
+
+```{group-tab} CLI
+
+You can share an authenticated session with another user by running:
+
+    anbox-stream-gateway share --description="Grant access to xxx"
+
+Running this command generates a presigned URL for the session, that is valid for a specified duration.
+
+You can later update the expiration date, description and the timeout values of a shared session:
```

Comment:
You can later update the expiration date, description and the timeout values of a shared session:
->
You can later update the expiration date, description of a shared session: